### PR TITLE
Release tiptap next

### PR DESCRIPTION
**Pull request by justame**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,15 +17,18 @@
     Click to see more.
   </summary>
 
+
+
+</details>
+<hr/>
+
+## 8.70.5 (Feb 21, 2022)
```
Comment (danarish4) [inline CHANGELOG.md:25]:
```suggestion
## 8.70.6 (Feb 21, 2022)
```

**Review by oded-soffrin — CHANGES_REQUESTED** [review]
wait for merge from master w/a fix for broken linter

**Comment by oded-soffrin** [comment]
> wait for merge from master w/a fix for broken linter

merged

**Review by oded-soffrin — COMMENTED** [review]
I never release a version from PR. that' cool!

**Pull request merged**